netweaver/monitoring.sls

```diff
@@ -0,0 +1,30 @@
+{%- from "netweaver/map.jinja" import netweaver with context -%}
+
+prometheus_sap_host_exporter_pkg:
+  pkg.installed:
+    - name: prometheus-sap_host_exporter
+
+sap_host_exporter_exporter_service:
+  service.running:
+    - name: prometheus-sap_host_exporter
+    - enable: True
+    - restart: True
+    - require:
+      - pkg: prometheus_sap_host_exporter_pkg
+      - file: sap_host_exporter_configuration
+
+# for each node the instance number change so we need to adapt configuration
```

Comment:
`... the instance number change**s**...`

Reply:
yeah always forgetting the s for 3 person :/ :grin: